pkg/haproxy/template.go

```diff
@@ -21,12 +21,13 @@ defaults
     timeout client          50000
     timeout server          50000
 
-{{ range . }}
-listen listen6-{{ .Port }}
-        bind	{{ .Source }}:{{ .Port }}
+{{ range $templ := . }}
+listen listen6-{{ $templ.ServicePort }}
+        bind	{{ $templ.Source }}:{{ $templ.ServicePort }}
         mode    tcp
-        server  dest4-{{ .Port }}    {{ .Dest }} send-proxy
         maxconn 28000
         grace   4000
```

Comment:
should we rip these out and just rely on the defaults until we understand what values we may want in here?

Reply:
I think that's a good idea. Some notes:

https://cbonte.github.io/haproxy-dconv/1.9/configuration.html#maxconn

Default 2000

https://cbonte.github.io/haproxy-dconv/1.9/configuration.html#grace

default unclear; can't find ATM